gm4_pulverizers/data/pulverizers/functions/chance/set_stack_count.mcfunction

```diff
@@ -0,0 +1,12 @@
+#@s = pulverizers containing a recipe results
+#Run from chance/calculate_chance
+
+execute if score result gm4_pv_chance matches ..50 if block ~ ~ ~ dropper{Items:[{Slot:8b,tag:{gm4_pulverizers:{chance:{id:"log"}}}}]} run scoreboard players add chance_stack gm4_stack_size 1
+execute if score result gm4_pv_chance matches ..20 if block ~ ~ ~ dropper{Items:[{Slot:8b,tag:{gm4_pulverizers:{chance:{id:"planks"}}}}]} run scoreboard players add chance_stack gm4_stack_size 1
+execute if score result gm4_pv_chance matches ..10 if block ~ ~ ~ dropper{Items:[{Slot:8b,tag:{gm4_pulverizers:{chance:{id:"blaze_rod"}}}}]} run scoreboard players add chance_stack gm4_stack_size 1
+execute if score result gm4_pv_chance matches ..40 if block ~ ~ ~ dropper{Items:[{Slot:8b,tag:{gm4_pulverizers:{chance:{id:"bone"}}}}]} run scoreboard players add chance_stack gm4_stack_size 1
+execute if score result gm4_pv_chance matches ..40 if block ~ ~ ~ dropper{Items:[{Slot:8b,tag:{gm4_pulverizers:{chance:{id:"sugar_cane"}}}}]} run scoreboard players add chance_stack gm4_stack_size 1
+execute if score result gm4_pv_chance matches ..20 if block ~ ~ ~ dropper{Items:[{Slot:8b,tag:{gm4_pulverizers:{chance:{id:"music_disc"}}}}]} run scoreboard players add chance_stack gm4_stack_size 2
```

Comment:
Music disc is the only exception here? That seems like a weird decision.

Reply:
It's an unintended coincidence. There used to be other recipes that would output more than one bonus item, but when I redid my list of pulverized items, the music discs ended up being the only one that had more than 1.
`music disc -> 3 flint + 20% 2 flint`